argcomplete/__init__.py

```diff
@@ -600,6 +600,17 @@ def display_completions(substitution, matches, longest_match_length):
         """
         return self._display_completions
 
+class ExclusiveCompletionFinder(CompletionFinder):
+    @staticmethod
+    def _action_allowed(action, parser):
+        class_name = ExclusiveCompletionFinder
+        allowed = super(class_name, class_name)._action_allowed(action, parser)
+
+        append_classes = (argparse._AppendAction, argparse._AppendConstAction)
+        not_exist = action not in parser._seen_non_default_actions or action._orig_class in append_classes
```

Comment:
Please rewrite this line. The variable name `not_exist` is unclear, please rename it. Please break up the logical statement into a more readable form.